integration_tests/macros/unit_tests/test_numeric_monitors.sql

```diff
@@ -12,10 +12,20 @@
     {%- set default_all_types = elementary.get_config_var('edr_monitors')['column_any_type'] | list %}
     {%- set default_numeric_monitors = elementary.get_config_var('edr_monitors')['column_numeric'] | list %}
     {% set numeric_monitors = [] %}
+
+
+    {# todo in here: ask someone who understands these unit tests to make sure if they are ran and how #}
+    {% set metric_properties = elementary.construct_metric_properties_dict(timestamp_column=none,
```

Comment:
I think it's generally preferable to take notes aside from the code.
What do you think?

Reply:
I wrote this so I don't push to master without doing it so it worked as intended 🤷 (Don't think it should be in master)

Do you know this unit test file? where it's called/ when?

Reply:
I'd check the CI, but I think it's mostly legacy at this point in time.

Reply:
Checking with Itamar

Reply:
Why not search references for it in the code though?